Lesson09_Task01.py

```diff
@@ -0,0 +1,36 @@
+import re
+"""
+Home-task 09 Files operations
+# Write script that will read "input.txt" file
+and find there all characters from english alphabet
+# collect these characters and their positions in file
+# after info collected this info as text
+write to "output.txt" file in such format
+# ####################
+# <first found letter> -> pos1
+# <next_letter> -> pos2
+# <next_letter -> pos3
+# etc
+# #######################
+"""
+
+# Read the input file
+with open('input.txt', 'r') as input_file:
+    # Read the contents of the file
+    content = input_file.read()
+
+# Find all English alphabet characters and their positions
+found_letters = re.findall(r"[A-Za-z]", content)
+letter_positions = [(char, content.index(char) + 1) for char in found_letters]
```

Comment:
this won't work for duplicate characters. Algorithm should be improved

Reply:
Updated the homework